src/db/config.ts

```diff
@@ -1,3 +1,3 @@
 export const port = '27017';
 export const host = '127.0.0.1';
-export const dbName = 'mern';
\ No newline at end of file
+export const dbName = 'acs';
```

Comment:
Missing a newline at the end of this file. Weird that eslint didn't catch this on save.
```suggestion
export const dbName = 'acs';

```

Reply:
Yea I we should add that to our eslint config. I don't think I have it